topi/python/topi/x86/conv2d.py

```diff
@@ -48,6 +60,7 @@ def _create_tuning_space(cfg, data, kernel, strides, padding, dilation, layout):
     sh, sw = strides if isinstance(strides, (tuple, list)) else (strides, strides)
     oh = (h - kh + 2 * ph) // sh + 1
     ow = (w - kw + 2 * pw) // sw + 1
+    cfg.add_flop(2*oh*ow*kh*kw*oc*ic)
```

Comment:
Shouldn't be necessary after https://github.com/dmlc/tvm/pull/2776, maybe add a TODO to remove?